sdk/cosmos/azure-cosmos/HISTORY.md

```diff
@@ -1,4 +1,45 @@
 # Change Log azure-cosmos
+
+## Version 4.0.0b2:
+
+Version 4.0.0b2 is the second iteration in our efforts to build a more Python client library.
+For more information about this, and preview releases of other Azure SDK libraries, please visit https://aka.ms/azure-sdk-preview1-python.
```

Comment:
This link points to "July 2019 Preview" will the content at this URL link be updated? Or does this URL need to be updated?

Reply:
Good point - we probably need a URL - I'll check